content/blog/18-19/state-of-the-wpi.md

```diff
@@ -0,0 +1,102 @@
+
+### OSM Worksops (14 en 28 november)
+
+Omdat open source/open data toch wel een belangrijk deel van Zeus is, organiseerden we samen met Pieter Vander Vennet een introductie tot de wereld van OpenStreetMap. In de eerste workshop leerden we mensen hoe ze zelf data toevoegen om zo de wereld een beetje verder in kaart te brengen. In de tweede leerden we werken met OsmAnd, de open routeplanner die draait op OpenStreetMap. We leerden hoe we OsmAnd kunnen 'hacken' om het zo te maken als we zelf willen.
+
+### Pannenkoeken en Jenever Codenight
+
+> **Ballmer-peak**: The theory that computer programmers obtain quasi-magical, superhuman coding ability when they have a blood alcohol concentration percentage between 0.129% and 0.138%.
+
+Ook deze keer sloten we het kalenderjaar af met een avond vol pannenkoeken, jenever en code. Dit jaar hebben we het ook gecombineerd met D-Ploy Day, wat inhield dat we zo veel mogelijk Zeusprojecten op zo veel mogelijk laptops lieten draaien om zo mensen te stimuleren om aan deze projecten te sleutelen. Zoals iedereen weet, Ruby wordt sneller gedeployed als men al een paar shotjes gedaan heeft, ,toch?
+
+### IHNGTWHSIPB-codenight (13 februari)
+
+Bij het zien van die naam denk je misschien "Wie heeft zijn toetsenbord laten vallen en dat als naam gekozen?", maar het is weldegelijk een afkorting. De "Ik Heb Nog Genoeg Tijd Want Het Semester Is Pas Begonnen"-codenight. Het tweede semester kwam op gang met een codenight gericht op iedereen die zich niet kon inhouden om te programmeren zonder dat er een deadline en punten boven het hoofd hing. Want wat doen Zeusers en Zeusinnen buiten de verplichte portie programmeren? Meer programmeren natuurlijk!
+
+### HTTPizza/2.0 (21 februari)
+
+Samen met IDLab organisserden we dit jaar de opvolger van HTTPizza in 2017. Met coole challanges in het thema van een pizzeria lieten we onze leden kennis maken met de wondere wereld van het web door onze eigen HTTP-variant. Een PIZZA-request sturen is toch zo veel leuker dan een GET-request?
+
+### Hashcode (28 februari)
+
+Ook dit jaar was Zeus van de partij op de Google Hashcode, de jaarlijkse challenge van Google waarbij teams uitgedaagd worden om een zo goed mogelijke oplossing te vinden voor een origineel probleem. We hadden dit jaar een team dat het tot de finale van de Hashcode gehaald heeft!
+
+### Capture The Friday (7 maart)
+
+Dit event viel zoals de naam al doet vermoeden op een vrijdag... Right? FOUT! Het was een donderdag! Hackers hebben vrijdag gestolen en het is aan ons om de dag te redden! Dit doen we door zoveel mogelijk dingen terug te hacken. Naast het mega-spannende verhaal was dit ook een super toffe capture the flag met challenges van verschillende moeilijkheden. Er werd ons ook nog eens duidelijk gemaakt waarom we geen PHP gebruiken.
+
+### 12urenloop (29 april)
```

Comment:
Misschien verwijzen naa de 12UL blogpost?

Reply:
Yes, maar die bestond nog niet toen ik deze blogpost geschreven had :D

Reply:
Ook, het was vooral over vorig jaar, dus denk ik dat een post over dit jaar niet helemaal relevant is...

Reply:
Ok :)